wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # if template is incomplete locally, delete and ask for a download
+    if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
Sorry, i missed this. 

`check_template` should only run for templates that the user downloaded online. These are the ones that may be incomplete because of a) lack of Internet connection or b) any interruption in the download process.